### DISCUSSION TITLE
module not found error

### DISCUSSION THREAD

**Discussion by Minamyle** [vercel/next.js#48363] [Help]
### Summary

i want to know why my server is giving me this error

### Additional information

```js
Compiled with problems:

ERROR in ./src/Components/Screens/HomeScreen.js 6:0-35
Module not found: Error: Can't resolve '../products' in 'C:\Users\Temitayo\OneDrive\Desktop\myleshop\frontend\src\Components\Screens'
```

**Comment by qpre** [comment]
it seems there is no "../products.js" or "../products/index.js" relative to your "HomeScreen.js" file.

It's near impossible to give you a better explanation than what is showed in your console without the actual code.
